README.md

```diff
@@ -22,6 +22,13 @@ In the European Union, this extension's fields correspond to [eForms BT-99 (Revi
         "url": "https://www.manchester.ac.uk/",
         "buyerProfile": "https://in-tendhost.co.uk/universityofmanchester/aspx/Home"
       }
+    },
+    {
+      "name": "Royal Tax Office",
+      "id": "08797655",
+      "roles": [
+        "informationService"
+      ]
```

Comment:
Let's add a contact point to this organization with the name, email, telephone previously used.

Reply:
Done in d9c9103f9e077c71be